blockexplorer/src/api/index.tsx

```diff
@@ -183,3 +184,29 @@ export const fetchBlocksTransactionsCChain =
     }
     return r;
   };
+
+
+  export async function loadValidatorsInfo() {
+    return new Promise((resolve, reject) => {
+      const urlValidators = `${getBaseUrl()}${baseEndpoint}/validatorsInfo`;
+      let networks = store.getState().appConfig;
+      let activeNetwork = networks.networks.find(
```

Comment:
active network is an unused variable i think this should be deleted, if the active network is needed you can select it directly from the store we are now using the same networks as the wallet and switch the active network in BE store to the one selected in the suite app.